agent/testagent.go

```diff
@@ -276,14 +277,6 @@ func (a *TestAgent) consulConfig() *consul.Config {
 	return c
 }
 
-func UniqueID() string {
```

Comment:
This is a pretty bogus function just used in unit tests. Let's delete this and have the tests pull in github.com/hashicorp/go-uuid and use a real unique ID instead.

Reply:
makes sense.

Reply:
Dropped `unique.ID` for `go-uuid`